docs/inference.rst

```diff
@@ -168,6 +168,19 @@ An example configuration file (named ``inference.ini``) is::
     ; polarization prior
     name = uniform_angle
 
+    ;
+    ;   Sampling transforms
+    ;
+    [sampling_parameters]
+    ; parameters on the left will be sampled in
+    ; parametes on the right
+    mass1, mass2 : mchirp, logitq
```

Comment:
Oops! That was a copy mistake, it should be q, not logitq. Thanks for catching, I've pushed a fix.